samples/nrf9160/asset_tracker/README.rst

```diff
@@ -68,7 +68,7 @@ Building and running
 
 This sample can be found under :file:`samples/nrf9160/asset_tracker` in the |NCS| folder structure.
 
-The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
+The sample is built as a non-secure firmware image for the nrf9160_pca10090ns board.
```

Comment:
The same instruction is also in the readme file for the secure boot sample.

Reply:
And should that readme also contain info like "build the sample for nrf9160_pca10090 and the application with nrf9160_pca10090ns"?

Reply:
I have addressed the first comment. Regarding the second, I don't think we need to document how to build the non-secure application in the `secure_boot` sample.

Reply:
But we do already - see line 49 in that readme (cannot comment there).

Reply:
Should be fixed.

Reply:
now it is correct